hs/Css/Graph.hs

```diff
@@ -330,6 +330,12 @@ sidebarCSS = do
         marginBottom (px 20)
         ":hover" & do
             backgroundColor grey2
+    ".flip" & do
+        "-webkit-transform" -: "scaleX(-1)"
```

Comment:
@christinem I think the `transform $ scaleX (-1)` was supposed to replace this entire block